db/fixtures/notification_types.yml

```diff
@@ -4,7 +4,7 @@
   :expires_in: 7.days
   :level: :success
   :audience: tenant
-- :name: service_provisioned
+- :name: automate_service_provisioned
```

Comment:
@gtanzillo @isimluk Note: These are being renamed so they do not generate notifications from the events.  We need more control over when to rise the notification.